config/vufind/searches.ini

```diff
@@ -290,7 +290,8 @@ WorkKeys = year
 ;       configure in the section of that file referenced by "ini section". Display
 ;       search results matching the terms found in the specified GET parameter
 ;       (default = "lookfor"), limited to a specified number of matches
-;       (default = 5).
+;       (default = 5). A result cache can be enabled in the [Cache_ExternalVuFind_Defaults]
```

Comment:
I'm not sure if it's accurate to say the cache can be enabled here, unless you disable it by default. I may be misremembering, but I'm pretty sure that if you use the caching downloader, you're going to have caching by default. It might also be worth reviewing the default ttl, etc., to be sure it's reasonable for this use case.

Reply:
Crap you are right, I had completely misread the defaults in config.ini (didn't realize I had a local override).  I've changed it to mirror the other [Cache]s, so enabled by default.  But I gave it a default ttl = 300, instead of infinite, as that mirrors the search cache default which I think is more applicable here.